x/axelarnet/keeper/grpc_query.go

```diff
@@ -38,8 +39,18 @@ func (q Querier) PendingIBCTransferCount(c context.Context, _ *types.PendingIBCT
 		if !ok {
 			return nil, fmt.Errorf("cosmos chain %s not found in the %s module", c, nexusTypes.ModuleName)
 		}
-		transfers := q.nexus.GetTransfersForChain(ctx, chain, exported.Pending)
-		counts[c.String()] = uint32(len(transfers)) // assert: there should never be more than 4294967295 transfers in the queue
+		pageRequest := &query.PageRequest{
+			Key:        nil,
+			Offset:     0,
+			Limit:      1,
+			CountTotal: true,
```

Comment:
What does `Limit` mean when `CountTotal` is set to `true`?

Reply:
Still the same as before, it'll process those amount of elements, but since count total is set, it'll also count the total number of elements. `0` will make it process a lot more (it'll use a default of `100`), so I'm keeping it to `1`

Reply:
Can you add a comment explaining that `Limit` being 1 and then merge?